pkgs/by-name/kt/ktor-cli/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "ktor-cli";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "ktorio";
+    repo = "ktor-cli";
+    rev = "refs/tags/${version}";
```

Comment:
```suggestion
    rev = version;
```

Reply:
No, using refs/tags/ is actually more correct.